pkgs/by-name/gh/ghostty/linux.nix

```diff
@@ -164,8 +155,6 @@ stdenv.mkDerivation (finalAttrs: {
     versionCheckHook
   ];
 
-  doInstallCheck = true;
```

Comment:
This disables versionCheckHook. It should not be removed

Reply:
Ah, the Nix documentation fails me. From what I read it didn't look like it did anything in our case.